### PR TITLE
Add a grafana dashboard to be deployed alongside the alerts

### PR DESCRIPTION
**Pull request by dazoakley**
I'll update this and the alerts in a future change to show the same
thresholds.

### CODE REVIEW

File: charts/generic-prometheus-alerts/templates/_helpers.tpl [hunk 1]
```diff
@@ -50,3 +50,25 @@ Selector labels
...
+*/}}
+{{- define "generic-prometheus-alerts.dashboardTags" -}}
+{{- .Release.Name | append .Values.extraDashboardTags | toJson -}}
+{{- end }}
+
+{{/*
+Labels for the grafana dashboard configmaps
+*/}}
+{{- define "generic-prometheus-alerts.dashboardLabels" -}}
+grafana_dashboard: ""
```
Comment (mattops) [inline charts/generic-prometheus-alerts/templates/_helpers.tpl:65]: does this need a value?

Comment (dazoakley) [reply charts/generic-prometheus-alerts/templates/_helpers.tpl:65]: Yeah unfortunately it looks like it doesn't get slurped into grafana without it 😞

Comment (dazoakley) [reply charts/generic-prometheus-alerts/templates/_helpers.tpl:65]: Ah, sorry, just realised I didn't answer your question - nope, it doesn't need a value, but it does need to be there in order for the loading to happen.

Comment (mattops) [reply charts/generic-prometheus-alerts/templates/_helpers.tpl:65]: yes, I just wondered if it was safer to set a value, even it's not used.

**Review by mattops — APPROVED** [review]

File: charts/generic-prometheus-alerts/templates/_helpers.tpl (hunk 1 above)
Comment (dazoakley) [reply charts/generic-prometheus-alerts/templates/_helpers.tpl:65]: I've not tried setting a value as yet - will have an experiment later to see if it makes a difference 👍

**Pull request merged**